<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerJupyterKernelModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerJupyterKernelModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="SetupCollapsibleButton">
     <property name="text">
      <string>Setup</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="AutomaticKernelInstallationGroupBox">
        <property name="title">
         <string>Automatic Slicer kernel installation: </string>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="PythonScriptsFolderLabel">
           <property name="toolTip">
            <string>Folder that contains 'jupyter-kernelspec' executable</string>
           </property>
           <property name="text">
            <string>Python Scripts folder:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ctkPathLineEdit" name="PythonScriptsFolderEdit">
           <property name="toolTip">
            <string>Folder that contains 'jupyter-kernelspec' executable</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="InstallSlicerKernelPushButton">
           <property name="text">
            <string>Install Slicer kernel in Jupyter</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="InstallSlicerKernelStatusLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Slicer kernel installation</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="KernelFolderLabel_2">
           <property name="text">
            <string>Run this command in a terminal in your Python environment:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="ManualInstallCommandTextEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CopyCommandToClipboardPushButton">
           <property name="text">
            <string>Copy command to clipboard</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="ControlCollapsibleButton">
     <property name="text">
      <string>Control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="StartJupyterNotebookPushButton">
        <property name="text">
         <string>Start Jupyter notebook</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
